### PR TITLE
Update conversation inbox visuals on mobile

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-1856]

Update the existing conversations view message preview to match the new inbox design. This ticket is focused on mobile only. Don’t worry about updating anything for the desktop layout. It’s okay for desktop to have a broken layout or a really stretched mobile layout. Don’t even worry about adding a fresnel xs wrapper.

__Before:__
Screen Shot 2020-04-15 at 2 12 02 PM

__After:__
Screen Shot 2020-04-15 at 1 18 51 PM

[PURCHASE-1856]: https://artsyproduct.atlassian.net/browse/PURCHASE-1856

### CODE REVIEW

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 1]
```diff
@@ -32,59 +39,78 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
...
+                    <Sans size="3" color={"black30"}>
+                      (message count)
+                    </Sans>
+                  </Flex>
+                  <Flex>
+                    <Sans size="3" mr="5px" color={"black30"}>
+                      <TimeSince
+                        time={conversation.lastMessageAt}
+                        size="3"
+                        exact
```
Comment (zephraph) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx:95]:
```suggestion
```

I don't think you want to use exact here.

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 2]
```diff
@@ -32,59 +39,78 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
...
+                      color={conversation.unread ? "black" : "black60"}
+                    >
+                      {partnerName}
+                    </Sans>
+                    <Sans size="3" color={"black30"}>
+                      (message count)
+                    </Sans>
+                  </Flex>
+                  <Flex>
+                    <Sans size="3" mr="5px" color={"black30"}>
```
Comment (zephraph) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx:91]: `TimeSince` is already a Sans component  that uses `black30`. I think you can delete this outer wrapper entirely and just pass the `mr` as a prop to that component.

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 3]
```diff
@@ -32,59 +39,78 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
...
-                <Serif size="2" ml={0.5} mr={0.5}>
-                  -
-                </Serif>
-                <Serif italic size="2" color="black60">
-                  {title && title.trim()}, {date}
-                </Serif>
-              </Flex>
-            </Link>
-            {conversationText}
+        <Flex alignItems="center" px={1} width="100%" height="120px">
```
Comment (zephraph) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx:62]: Is the `height` needed here?

File: src/Apps/Artist/Routes/AuctionResults/Components/ImageWithFallback.tsx [hunk 4]
```diff
@@ -8,3 +9,18 @@ export const ImageWithFallback = ({ Fallback, ...props }) => {
...
+export const renderFallbackImage = () => {
+  return (
+    <Box bg="black5" width="100%" height="100%">
+      <Flex
+        alignItems="center"
+        justifyContent="center"
+        width="100%"
+        height="100%"
+      >
+        <NoArtworkIcon width="28px" height="28px" fill="black30" />
```
Comment (zephraph) [inline src/Apps/Artist/Routes/AuctionResults/Components/ImageWithFallback.tsx:22]: It's worth noting that the icon is probably going to be much smaller than the size of our images here. Might be worth trying to break one of the URLs and see how it looks.

Comment (lilyfromseattle) [reply src/Apps/Artist/Routes/AuctionResults/Components/ImageWithFallback.tsx:22]: The icon is meant to be small, it sits in the middle of a grey box where the image should be. But testing is a good idea regardless.

File: src/Apps/Conversation/Components/ConversationSnippet.tsx (hunk 3 above)
Comment (lilyfromseattle) [reply src/Apps/Conversation/Components/ConversationSnippet.tsx:62]: Its so I can center the items inside it vertically

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.20.0`](https://github.com/artsy/reaction/releases/tag/v26.20.0) :rocket:

**Pull request merged**